index.html

```diff
@@ -329,7 +329,9 @@ <h5>CIRCT</h5>
             depending on your machine.
             Also, when configuring CIRCT with cmake (step 4), make
             sure to set the
-            flag <code>-<nobr>DCIRCT_SLANG_FRONTEND_ENABLED=ON</nobr></code>. 
+            flag <code>-<nobr>DCIRCT_SLANG_FRONTEND_ENABLED=ON</nobr></code>.</br>
+	    <b>Note:</b> Make sure to get the commit "f1213ba8bdbcaba2a4903c5e078fb8c223a2ab39"
+	    as this fixes an issue with the CIRCT build.
```

Comment:
```suggestion
	    <b>Note:</b> Make sure to get the commit "f1213ba8bdbcaba2a4903c5e078fb8c223a2ab39"
	    by simply updating the cloned repository using <code>git pull</code>
	    or cloning a fresh version of the repository, as this fixes an issue with
	    the CIRCT build.
```
Maybe something like this makes it a bit more clear for people who are not that used to using Git?